src/query/service/src/pipelines/processors/transforms/window/transform_window.rs

```diff
@@ -225,6 +225,7 @@ impl<T: Number> TransformWindow<T> {
 
         let partition_by_columns = self.partition_indices.len();
         if partition_by_columns == 0 {
+            self.partition_size += self.block_rows(&self.partition_end);
```

Comment:
This only effects percent rank? If so maybe should configure current function is percent rank

Reply:
partition_size is a important part of the context of the current partition,
no only percent rank use it.